packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -38,8 +39,21 @@ export class JaegerExporter implements SpanExporter {
       typeof config.flushTimeout === 'number' ? config.flushTimeout : 2000;
 
     config.host = config.host || process.env.JAEGER_AGENT_HOST;
+    config.endpoint = config.endpoint || process.env.JAEGER_ENDPOINT;
+    config.username = config.username || process.env.JAEGER_USER;
+    config.password = config.password || process.env.JAEGER_PASSWORD;
```

Comment:
Done in https://github.com/open-telemetry/opentelemetry-js/pull/965/commits/7e645661f87936eddc582109bb3ff7b6d9a76fca

Reply:
With your commit, i think it's not possible to only use `JAEGER_AGENT_ENDPOINT` env var since you check for `config.endpoint` which should be undefined in this case. Is that normal ?

Reply:
If `process.env.JAEGER_ENDPOINT` is defined, then `config.endpoint` will also be. I don't think i understood what you meant.